docs/meetings/backend/Meeting 018.md

```diff
@@ -0,0 +1,11 @@
+# Backend team meeting #18
+
+When: 3/3/21, 21:00 UTC
+Attending: Aapeli, Andrea, Darren, Lucas, Sam
+
+## Notes
+
+* Events
+* Regions visited/lived
+* User blocking
+* Profile picture stuff still buggy: no way to 
```

Comment:
Is this symbolic, like "No way to make profile pictures blank" but the sentence is blank?